### PR TITLE
Fix TS

### PR DESCRIPTION
**Pull request by junderw**

### CODE REVIEW

File: src/core/cryptography.ts [hunk 1]
```diff
@@ -11,14 +11,14 @@ export class Cryptography {
...
-    buf: Buffer | string | elliptic.ec.KeyPair
+    buf: Buffer | string | elliptic.ec.KeyPair,
   ): elliptic.ec.KeyPair {
     return ec.keyFromPrivate(buf);
   }
 
   public static get_sin_from_key(kp: elliptic.ec.KeyPair): string {
     const pk: crypto.BinaryLike = Buffer.from(
-      kp.getPublic().encodeCompressed()
+      kp.getPublic().encodeCompressed(),
```
Comment (NicolasDorier) [inline src/core/cryptography.ts]: what is this `,` ??

File: src/core/cryptography.ts [hunk 2]
```diff
@@ -11,14 +11,14 @@ export class Cryptography {
   }
 
   public static load_keypair(
-    buf: Buffer | string | elliptic.ec.KeyPair
+    buf: Buffer | string | elliptic.ec.KeyPair,
```
Comment (NicolasDorier) [inline src/core/cryptography.ts]: what is this `,` ??

**Comment by NicolasDorier** [comment]
You added a bunch of trailing `,`, what is this?

File: tsconfig.json [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
   "compilerOptions": {
+    "target": "es2017",
     "module": "commonjs",
-    "esModuleInterop": true,
-    "target": "es6",
-    "moduleResolution": "node",
-    "sourceMap": true,
+    "strict": true,
+    "declaration": true,
     "outDir": "dist",
```
Comment (takinbo) [inline tsconfig.json]: ```outDir``` specifies ```dist``` as the destination. Isn't this where the ```index.js``` file will go?

**Comment by junderw** [comment]
TypeScript/JavaScript syntax allows trailing commas, and prettier is set to use trailing commas to make it easier to read diffs.

(Many times lines are marked as diff only because you added an item after and in order to do that you had to add a comma to the previous line)

**Comment by NicolasDorier** [comment]
Ping @Uter1007 @takinbo I think this PR is awesome. It added types and now @takinbo can just publish to npm by bumping the version and running `npm publish`.

**Comment by NicolasDorier** [comment]
ACK for me!

**Comment by takinbo** [comment]
@NicolasDorier actually we never published to npm :smile:. Instructions are to install from GitHub rather than npm.

**Comment by junderw** [comment]
@takinbo ahhhh that is why you have problems lololol...

using prepare script or postinstall script will run after npm install.

So if you npm install from github with this PR it will work fine now.

**Comment by takinbo** [comment]
@junderw awesome! It's an ACK for me as well. Are there any other commits you want to submit?

**Comment by junderw** [comment]
I made npm test check format and lint, it will fail if either check fails... and it echos a warning.

**Comment by junderw** [comment]
Should be good now

**Comment by NicolasDorier** [comment]
@takinbo I just create a npm account. Don't hesitate to push the node-btcpay package. After can you give me permissions ? My account is `nicolas.dorier`.

**Comment by takinbo** [comment]
@NicolasDorier the package is not on npm and can be installed directly from GitHub but if you think we should upload it to npm, I'll also create an account and do so.

**Comment by NicolasDorier** [comment]
@takinbo yeah, it should be on npm, that's how the cool kids are doing.

**Pull request merged**